src/OrbitGl/TriangleToggle.cpp

```diff
@@ -34,31 +34,32 @@ void TriangleToggle::DoDraw(PrimitiveAssembler& primitive_assembler, TextRendere
   Color color = is_collapsible_ ? kWhite : kGrey;
 
   // Draw triangle.
-  static float half_sqrt_three = 0.5f * sqrtf(3.f);
```

Comment:
I don't understand. The size of the triangle didn't seem to change after collapsing, why are you now basing these calculations in the state? Do you eventually think on basing the size of triangle in which state it is?

Reply:
I'm not sure what's the best expected behavior here tbh. As of now it doesn't matter because width and height are set to the same value, but this is not enforced by the TriangleToggle itself. *If* width any height would be set to different values then yes, the current calculations would change the size of the triangle depending on the state. This is weird, but it is also weird to have the clicking area differ extremely from what's actually drawn (because the picking rendering uses the full width / height of the element). So I decided to at least make it consistent.

Reply:
I could also just remove the `setHeight` method and enforce height == width this way, wdyt?

Reply:
I don't like. I would rather force a consistent size by using the minimum between width() and height() to choose the base_size.

Reply:
I'm also ok with removing setHeight()

Reply:
I've thought this over and I think removing `setHeight` is a bit confusing because `setWidth` would then be used to set both width and height. I've changed it to act as you suggested, it's now using the smaller of both sides and centers the triangle. PTAL.